data/mods/Magiclysm/monsters/forgedwellers.json

```
@@ -66,6 +68,7 @@
     "copy-from": "mon_gracke",
     "special_attacks": [ { "type": "spell", "spell_data": { "id": "eshaper_shardspray", "min_level": 5 }, "cooldown": 10 } ],
```

```suggestion
    "aggression": 0,
    "special_attacks": [ { "type": "spell", "spell_data": { "id": "eshaper_shardspray", "min_level": 5 }, "cooldown": 10 } ],
```
In case that for Gracken is changed.

---

Won't that cause a field not changed from copy-from error?